sql/testdata/builtin_function

```diff
@@ -219,3 +219,89 @@ query T
 SELECT right('abcde', 2)
 ----
 de
+
+# TIMESTAMP/DATE builtins.
+
+query T
+SELECT age(timestamp '2001-04-10 22:06:45', timestamp '1957-06-13')
+----
+384190h6m45s
+
+query T
+SELECT age(timestamp '1957-06-13') - age(now(), timestamp '1957-06-13') < interval '1s'
+----
+true
+
+query T
+SELECT now() - timestamp '2015-06-13' > interval '100h'
+----
+true
+
+query T
+SELECT now() - clock_timestamp() < interval '1s' 
+----
+true
+
+query T
+SELECT now() - current_date < interval '24h1s'
```

Comment:
regardless, let's increase the margin of error (or otherwise make this resilient in the face of slow machines)